.github/workflows/main.yml

```
@@ -345,17 +188,15 @@ jobs:
   build-plugin-staging-pr:
     name: Build and Deploy Plugin
     needs:
-      - build-api
-      - build-web
-      - build-unraid-ui-webcomponents
+      - build-artifacts
       - test-api
     uses: ./.github/workflows/build-plugin.yml
     with:
       RELEASE_CREATED: false
```

actionlint (1.7.8)

input "RELEASE_CREATED" is typed as string by reusable workflow "./.github/workflows/build-plugin.yml". bool value cannot be assigned

(expression)